build.cmd

```diff
@@ -152,6 +152,14 @@ set Platform=
 if NOT "%__DotNetCliPath%" == "" goto SetupManagedBuild
 
 set "__DotNetCliPath=%__RootBinDir%\tools\cli"
+if "%__CleanBuild%"=="1" (
```

Comment:
If packages dont run into conflicts, we can keep them. But, semantically, a clean build should clean everything. We could start off with "clean" only cleaning the "bin" dir but keeping the packages.